### ISSUE TITLE
Not able to open a PDF with security - Adobe opens it fine without password entry

### ISSUE THREAD

**Issue by gbarnabic** [py-pdf/PyPDF2#249]
I have an Social Security pdf document that has Document Security as:
  Document Open Password: No
  Permissions Password: Yes

I tried this code thinking all I needed was to decript with a blank password, but it doesn't work:

```python
from PyPDF2 import PdfReader

reader = PdfReader("ssa-44.pdf")
```

![pdf properties](https://cloud.githubusercontent.com/assets/4430162/13239797/1032f704-d9ab-11e5-8556-5b2ec90ddd3f.png)

**Comment by gbarnabic** [comment]
[ssa-44.pdf](https://github.com/mstamy2/PyPDF2/files/141817/ssa-44.pdf)

Here is the file I'm trying to open.

**Comment by gbarnabic** [comment]
More research on here and I see that this issue has already been reported #53 (NotImplementedError("only algorithm code 1 and 2 are supported")) and may be addressed by claird (aka Selma) gets some time on his hands to work on PyPDF2 again.  

I did look at the code referenced there [link](https://github.com/mozilla/pdf.js/blob/master/src/core/crypto.js) and it is somewhat above my abilities to port over.  If the US Social Security Administration is using the 128-bit AES encryption level it may become more of an issue going forward. Especially with all the other security and encryption discussions going on of late.

**Comment by MartinThoma** [comment]
I think this got solved with #1015
